packages/ui/useScroll/useScroll.js

```diff
@@ -1070,11 +1070,12 @@ const useScrollBase = (props) => {
 			}
 		}
 
+		if (scrollContentHandle.current && scrollContentHandle.current.setScrollPositionTarget) {
+			scrollContentHandle.current.setScrollPositionTarget(targetX, targetY);
```

Comment:
Could you change the description above setScrollPositionTarget function.

// scrollMode 'translate' only
	setScrollPositionTarget (x, y) {